cluster-install/install.sh

```diff
@@ -842,7 +885,8 @@ make_astra_control_request() {
     fi
 
     logdebug "$method --> '$url'"
-    local -r result="$(curl -X "$method" -s $skip_tls_validation_opt -w "\n%{http_code}" "$url" "${headers[@]}")"
+    local -r result="$(curl -X "$method" -sS $skip_tls_validation_opt -w "\n%{http_code}" "$url" "${headers[@]}" 2> "$__ERR_FILE")"
```

Comment:
This is the new way of checking if images exist; we directly fetch the manifest, which is much faster than the previous method of listing tags (especially when there's a lot of tags) when the manifest exists -- something like going from a full minute difference for some images on `docker.repo.eng.netapp.com` to practically instant.

Unfortunately, if the manifest is missing, it takes almost as long as it did with the previous method, for some reason. Still much better though since, in most cases, the manifests _will_ exist.

This is also the first application of the `__ERR_FILE` "pattern". The output of **stdout** gets captured into the `result` variable, while `stderr` is redirected into the __ERR_FILE. We then read the contents of __ERR_FILE on the next line by calling `get_captured_err`, which will **cat** the value and then delete the file.